hw/ip/kmac/data/kmac.hjson

```diff
@@ -251,12 +251,12 @@
       ]
     } // R: STATUS
     { multireg: {
-        name: "KEY"
+        name: "KEY_SHARE0"
         desc: '''KMAC Secret Key
 
               KMAC secret key can be up to 512 bit.
               Order of the secret key is:
-              key[512:0] = {KEY0, KEY1, KEY2, ... , KEY15};
+              key[512:0] = {KEY15, KEY14, ... , KEY0};
 
               The registers are allowed to be updated when the engine is in Idle state.
               If the engine computes the hash, it discards any attempts to update the secret keys
```

Comment:
Would defining a regwen bit be simpler?

Reply:
I've also considered that... The history was:
1. Key to be stored as masked form, so the key written to the KMAC needs to go through masking logic (at this time assumed key stored in FW unmasked)
2. After discussed with SW ppl, looks like masked registers are better, so just added one more masked register `KEY_SHARE1` here. and... KMAC core logic already has secret key implemented, so just kept `hwext` as true :)